devicetypes/smartthings/smartsense-moisture-sensor.src/smartsense-moisture-sensor.groovy

```diff
@@ -223,6 +223,7 @@ private Map getBatteryResult(rawValue) {
 	else {
 		if (volts > 3.5) {
 			result.descriptionText = "{{ device.displayName }} battery has too much power: (> 3.5) volts."
+			result.value = 100
```

Comment:
I'm not sure there is any benefit of having this custom descriptionText. I think you should just remove this if statement and let the regular voltage handling take care of the conversion.